### ISSUE TITLE
pass keychain doesn't use correct path

### ISSUE THREAD

**Issue by braderhart** [Versent/saml2aws#440]
I am using the default pass keychain for Linux. I have an environment variable set `PASSWORD_STORE_DIR` but `saml2aws` doesn't recognize it when configuring my user. I have to unset it and initialize a new password store at the default path.

**Comment by runultra** [comment]
@braderhart did you happen to find any permanent resolution to this?

**Comment by braderhart** [comment]
No, except to initialize a store at the default directory just for saml2aws.
